app/javascript/react/components/RestaurantIndex.js

```diff
@@ -0,0 +1,46 @@
+
+import React, { useEffect, useState } from 'react'
+import RestaurantTile from './RestaurantTile'
+
+const RestaurantIndex = (props) => {
+  const [restaurantData, setRestaurantData] = useState([])
+
+   useEffect(() => {
+    fetch('/api/v1/restaurants?location=boston')
+    .then(response => {
+      if (response.ok) {
+        return response
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage);
+        throw (error);
+      }
+    })
+    .then(response => response.json())
+    .then(responseBody => {
+      setRestaurantData(responseBody)
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`))
+   }, [])
+
+  const restaurantTileArray  = restaurantData.map((restaurant) => {
+      
+    return(
+      <div>
+        <RestaurantTile
+          name={restaurant.name}
+          id={restaurant.id}
+          key={restaurant.id}
+        />
+      </div>
```

Comment:
oops, you shouldnt ever have a surrounding JSX element (in this case a `div`) around a component when doing this map. If you want each tile to have a div, you should move that div INTO the Restaurant tile. 

Otherwise, Im sure if I walked over I would see the "each element should have a key" error in your console right now